recipes/nodejs_from_binary.rb

```diff
@@ -47,6 +47,7 @@
 binaries = ['bin/node']
 
 binaries.push('bin/npm') if node['nodejs']['npm']['install_method'] == 'embedded'
+binaries.push('bin/npx') if node['nodejs']['npm']['install_method'] == 'embedded'
```

Comment:
You'll understand your intent much better than I will.  Comment though, does it make sense to use node['nodejs']['npm']['install_method'] instead of say node['nodejs']['npx']['install_method'].  I'm really asking.  It seems like node['nodejs']['npm']['install_method'] is a bit overloaded.  Thanks

Reply:
@MarkGibbons I _think_ my original intent here was to couple the two together, since `npx` is not useful without `npm`. Also, I think `npx` is installed alongside `npm` now?

All that said, yes, we can definitely make this more flexible if that seems worthwhile.

Reply:
Yep, now that I’ve read through things again, that was my intent. The `npx` bin comes with `npm`, and, in a manual install situation, you’d end up with both binaries. So we avoid variable/attribute sprawl with the above.